www/index.js

```diff
@@ -152,16 +159,16 @@ HotPush.prototype._hasloadedLocalFile = function() {
 HotPush.prototype.update = function() {
   var self = this;
   if (this.options.type === 'replace') {
-    this._syncs = [ContentSync.sync({ src: this.options.archiveURL, id: 'assets', headers: this.options.headers})];
+    this._syncs = [ContentSync.sync({ src: this.options.archiveURL, id: 'assets', copyCordovaAssets: true, headers: this.options.headers})];
```

Comment:
Hum we are using it with react also and it's working just by using `hotpushes.loadAllLocalFiles()` but i'm using the `documentsPath` option (because of WKWebview), that's maybe why